Engine Hacks/Strmag/Str Mag Split/Autolevelling and Saves/Autolevel and Save EA.event

```diff
@@ -8,13 +8,13 @@
 	//Write bases to ram
 	ORG $17E5C
 	callHack_r3(Write_Bases_to_RAM)
-	
+	/*
```

Comment:
Is this also a fix?

Reply:
This and the ending block comment completely remove the `Char_Mag_Autolevel` hook which seems to mess up the magic autolevel routine. The only hack necessary should be `Class_Mag_Autolevel`. I'm unsure why `Char_Mag_Autolevel` was included or who added it, but it was before I started working on this. What it seems to do is add character magic autolevel in the class autolevel routine while, so generics would have way too much magic. I checked, and character magic in autolevel should already be accounted for in its own routine if any of that makes sense.